.github/workflows/plugin-deploy.yml

```diff
@@ -0,0 +1,61 @@
+name: Deploy plugin to Azure function app
+
+on:
+  workflow_call:
+    secrets:
```

Comment:
I would not have added this `secrets` block myself. Can you explain why it's here? (Trying to work out if it's something I _should_ be  doing lol)

Reply:
I think some of these should be inputs, not secrets. See: https://github.com/gliff-ai/.github/blob/e9fab1e3f6b81c5be9af93c24e0a4725c6310e4b/.github/workflows/dependency-updates.yml#L4

Reply:
oh, I didn't know of "inputs".. makes sense, given that some of the vars are not real secrets, but just data passed to the action.. I liked to the idea of changing a plugin without having to change the code, but possibly not what we want given that changes in secrets are not versioned.